core/src/components/cat-tag/cat-tag.tsx

```diff
@@ -119,6 +119,11 @@ export class CatTag {
    */
   @Prop() tagCreationChars: string[] = [' '];
 
+  /**
+   * Whether the catChange event will be emitted when the input is blurred.
```

Comment:
Bit of misleading docs here. It's not about the catChange event. It's more general the adding on blur

Reply:
updated